exercises/practice/diamond/test/diamond_test.dart

```diff
@@ -1,120 +1,87 @@
 import 'package:diamond/diamond.dart';
 import 'package:test/test.dart';
 
-final diamond = Diamond();
-
 void main() {
+  final diamond = Diamond();
+
   group('Diamond', () {
-    test('Degenerate case with a single "A" row', () {
-      final List<String> result = diamond.rows('A');
-      expect(
-        result,
-        equals([
-          'A',
-        ]),
-      );
+    test('Degenerate case with a single \'A\' row', () {
+      final result = diamond.rows('A');
+      expect(result, equals(<String>['A']));
     }, skip: false);
 
     test('Degenerate case with no row containing 3 distinct groups of spaces', () {
-      final List<String> result = diamond.rows('B');
-      expect(
-        result,
-        equals([
-          ' A ',
-          'B B',
-          ' A ',
-        ]),
-      );
+      final result = diamond.rows('B');
+      expect(result, equals(<String>[' A ', 'B B', ' A ']));
```

Comment:
@kytrinyx Do note we have the list within `equals` formatted in a way so it's displayed as a diamond. Since the exercise is about generating these diamond string, I think we would like to maintain that. So it does require manual change.

Reply:
Good call. I've updated these to have the original diamond shape.